src/components/styles/Layout.tsx

```diff
@@ -63,3 +63,12 @@ export const InputLabel = styled.label`
 export const ErrorMessage = styled(Body)`
   color: ${uiColors.red.base};
 `;
+
+export const HR = styled("hr")`
+  background-color: ${uiColors.gray.light2};
```

Comment:
nit. Can we use destructuring for this. `background-color: ${gray.light2}`

Reply:
I prefer to keep this as to avoid an additional assignment that will only be used once in this file.